tests/templates/test_state_preparations/test_qutrit_basis_state_prep.py

```diff
@@ -0,0 +1,218 @@
+# Copyright 2018-2021 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the QutritBasisStatePreparation template.
+"""
+# pylint: disable=too-many-arguments
+import pytest
+import numpy as np
+import pennylane as qml
+
+
```

Comment:
In the tests it seems like most of the parametrizations of `basis_state` are composed of only 0s and 1s. Could you add additional parametrizations that include the 2nd excited state?

Reply:
done!